apps/site/pages/en/learn/getting-started/security-best-practices.md

```diff
@@ -449,6 +449,7 @@ The [OpenSSF][] is leading several initiatives that can be very useful, especial
 [typosquatting]: https://en.wikipedia.org/wiki/Typosquatting
 [Mitigations for lockfile poisoning]: https://blog.ulisesgascon.com/lockfile-posioned
 [`npm ci`]: https://docs.npmjs.com/cli/v8/commands/npm-ci
+[`npm-audit`]: https://docs.npmjs.com/cli/v9/commands/npm-audit
```

Comment:
unless the version is important (the text suggests not), removing the version will default it to latest, but leaving it there will bring the visitor to outdated data.

if people approve this suggestion, we should comb through all the docs for this pattern

```suggestion
[`npm-audit`]: https://docs.npmjs.com/cli/commands/npm-audit
```

Reply:
fixed these references via https://github.com/nodejs/nodejs.org/commit/39855a75a4b382d0fbc39c1f3b611860b91b12f7 as part of troubleshooting something else